test/quantization/fx/test_quantize_fx.py

```diff
@@ -5263,6 +5263,50 @@ def forward(self, x):
         # make sure it runs
         m(*example_inputs)
 
+    def test_channel_shuffle_lowering(self):
+        # Three versions of channel shuffle
+        class M1(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.op = torch.nn.ChannelShuffle(2)
+
+            def forward(self, x):
+                return self.op(x + x) + x
+
+        class M2(torch.nn.Module):
+            def forward(self, x):
+                return torch.channel_shuffle(x + x, 2) + x
+
+        class M3(torch.nn.Module):
+            def forward(self, x):
+                return torch.nn.functional.channel_shuffle(x + x, 2) + x
+
+        x = torch.randn(4, 4, 4, 4)
+        models = [M1().eval(), M2().eval(), M3().eval()]
+        # torch.channel_shuffle is torch.nn.functional.channel_shuffle
+        expected_nodes = [
+            ns.call_module(torch.nn.ChannelShuffle),
+            ns.call_function(torch.channel_shuffle),
+            ns.call_function(torch.channel_shuffle)
+        ]
```

Comment:
nit: might be better to just write a list of tuple of (model, expected_node) so that it's clearer

Reply:
OK. It's done. Please take a look.